i18n/tr.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Is this still work in progress?

Reply:
oh, I missed this file portion. I will be add the missing translation, ASAP.

Reply:
I added  the missing translated files.